src/components/Divider/Divider.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import styled, { StyledComponent } from 'styled-components';
+import { compose, space, SpaceProps } from 'styled-system';
+import { theme } from '../../essentials/theme';
+import { get } from '../../utils/themeGet';
+
+type DividerOffset = number | string;
+
+interface DividerProps extends SpaceProps {
+    /**
+     * Set the direction of the divider to vertical
+     */
+    vertical?: boolean;
+    /**
+     * Set offset / margin of the divider from the surrounding content
+     */
+    offset?: DividerOffset;
+}
+
+const HorizontalLine: StyledComponent<'div', typeof theme, DividerProps, 'theme'> = styled.div.attrs({ theme })<
+    DividerProps
+>`
+    width: 100%;
+    margin: 0 auto;
+    border: 0;
+    border-top: 0.06rem solid ${get('semanticColors.border.primary')};
+
+    ${compose(space)}
+`;
+
+const VerticalLine: StyledComponent<'div', typeof theme, DividerProps, 'theme'> = styled.div.attrs({ theme })<
+    DividerProps
+>`
+    display: inline-block;
+    width: 0.06rem;
+    margin: 0 auto;
+    border: 0;
+    border-left: 0.06rem solid ${get('semanticColors.border.primary')};
```

Comment:
We agreed to change it to `1px` because the divider shouldn't scale with font size